src/components/OrgListCard/OrgListCard.tsx

```diff
@@ -42,18 +42,20 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
       <div className={styles.orgCard}>
         <div className={styles.innerContainer}>
           <div className={styles.orgImgContainer}>
-            {image ? (
-              <img
-                src={image}
-                className={styles.orgimg}
-                alt={`${name} image`}
-              />
-            ) : (
-              <div
-                className={styles.emptyImg}
-                data-testid="emptyContainerForImage"
-              />
-            )}
+            <img
+              src={
+                image
+                  ? image
+                  : `https://api.dicebear.com/5.x/initials/svg?seed=${name
+                      .split(/\s+/)
+                      .map((word) => word.charAt(0))
+                      .slice(0, 2)
+                      .join('')}`
+              }
+              className={styles.orgimg}
```

Comment:
Add a border radius of 4px

Reply:
The border radius is already set to 4px for `orgImgContainer` class. Should I increase it?
I removed `className={styles.orgimg}` due to the absence of corresponding CSS styles.

Reply:
The border radius on orgImgContainer class will not work unless this property is applied to it, 
```css
overflow: hidden;
```
better directly apply border-radius to image, remove the unused CSS too, we don't want any bloat in our code.

Reply:
Done.
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/49762303/85d0f96e-b375-4509-a988-84f4dcd47c7e)

Reply:
Done
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/49762303/447038ec-aeed-4af9-bd0d-f9cba1186986)